AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -1579,6 +1736,22 @@ protected override void OnKeyDown(KeyEventArgs e)
                         Rotate(CurrentObjects);
                     }
                     break;
+                case Key.Z:
+                    if (IsControlPressed() && IsShiftPressed())
```

Comment:
Control + Shift + Z for redo?

Reply:
It's quite common in a lot of IDEs/things that I use so thought it was worth including as well.